cmd/sovereignnode/sovereignNodeRunner.go

```diff
@@ -1376,6 +1381,7 @@ func (snr *sovereignNodeRunner) CreateManagedProcessComponents(
 		OutGoingOperationsPool:                outGoingOperationsPool,
 		DataCodec:                             dataCodec,
 		TopicsChecker:                         topicsChecker,
+		OperationsHasher:                      outgoingOperationsHasher,
```

Comment:
the args below RunTypeComponents should be maybe moved to runTypeComponents in a future cleanup&refactor PR.